Dockerfile

```diff
@@ -183,7 +185,15 @@ RUN fname="go-ipfs_${IPFS_VERSION}_${TARGETOS:-linux}-${TARGETARCH:-amd64}.tar.g
   tmpout=$(mktemp -d) && \
   tar -C ${tmpout} -xzf "${fname}" go-ipfs/ipfs && \
   mv ${tmpout}/go-ipfs/ipfs /out/bin/ && \
-  echo "- IPFS: ${IPFS_VERSION}" >> /out/share/doc/nerdctl-full/README.md
+  echo "- IPFS: ${IPFS_VERSION}" >> /out/share/doc/nerdctl-full/README.md \
+ARG COSIGN_VERSION
+RUN fname="cosign-${TARGETOS:-linux}-${TARGETARCH:-amd64}" && \
+  curl -o "${fname}" -fSL "https://github.com/sigstore/cosign/releases/download/${COSIGN_VERSION}/${fname}" && \
```

Comment:
404

Reply:
it says so because it seems that COSIGN_VERSION is not replaced in the URL, why is that? I've defined it as an ARG with a value of v1.4.1.

Reply:
did I do something wrong?

Reply:
The URL that I'm trying to construct is correct:
* fname=cosign-linux-amd64
* COSIGN_VERSION=v1.4.1
* https://github.com/sigstore/cosign/releases/download/${COSIGN_VERSION}/${fname}

👉 https://github.com/sigstore/cosign/releases/download/v1.4.1/cosign-linux-amd64

Reply:
kindly ping 🙋🏻‍♂️

Reply:
Maybe try to echo the same string in your Dockerfile to say if it is expected.